mapboxgl/style_nolabel_tpl.json

```
@@ -0,0 +1,4122 @@
+{
+  "version": 8,
+  "name": "@style_name@",
```

Maybe we want to change this to:
```suggestion
  "name": "@style_name@ without labels",
```